app/controllers/pxe_controller.rb

```diff
@@ -54,7 +55,13 @@ def explorer
     render :layout => "application"
   end
 
-  private
+  def title
+    "PXE"
```

Comment:
Oh, one more thing, it wasn't there before, but we should really use gettext here too.. so `_("PXE")` please :)